src/qml/About.qml

```diff
@@ -122,21 +122,30 @@ Item {
             Layout.fillWidth: true
             Layout.maximumWidth: parent.width
             Layout.alignment: Qt.AlignCenter
+            Layout.bottomMargin: 10
             horizontalAlignment: Text.AlignHCenter
             font: Theme.tinyFont
-            color: Theme.light
-            opacity: 0.6
+            color: Theme.secondaryTextColor
+            textFormat: Text.RichText
 
             text: {
-                var dataDirs = platformUtilities.appDataDirs();
+                let label = '';
+                let isDesktopPlatform = Qt.platform.os !== "ios" && Qt.platform.os !== "android";
+                let dataDirs = platformUtilities.appDataDirs();
```

Comment:
```suggestion
                const isDesktopPlatform = Qt.platform.os !== "ios" && Qt.platform.os !== "android";
                const dataDirs = platformUtilities.appDataDirs();
```
I know it is a backport, but I feel obliged to comment :)

Reply:
@suricactus , well received, I'll fix it on master :)